packages/eslint-plugin-pf-codemods/test/rules/v5/spinner-remove-issvg.js

```diff
@@ -1,14 +1,11 @@
 const ruleTester = require('../../ruletester');
-const rule = require('../../../lib/rules/v5/spinner-svg-default');
+const rule = require('../../../lib/rules/v5/spinner-remove-issvg');
 
-ruleTester.run("spinner-svg-default", rule, {
+ruleTester.run("spinner-remove-isvg", rule, {
```

Comment:
```suggestion
ruleTester.run("spinner-remove-isSvg", rule, {
```

Reply:
oops... missed an s. for the sS i'll have to change the filenames as well though